packages/alchemy/src/react/components/auth/card/passkey-added.tsx

```diff
@@ -0,0 +1,18 @@
+import { PasskeyIllustration } from "../../../icons/passkey.js";
+import { PoweredBy } from "../../poweredby.js";
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+export function PasskeyAdded() {
```

Comment:
General question any reason why we'd want to disable jsdocs here? I think for anything we export publically it's probably worth adding some version of jsdocs.

Reply:
don't think this should be used publicly since its just a random subcomponent 🤔